### DISCUSSION TITLE
I want to learn! Any suggestions!?

### DISCUSSION THREAD

**Discussion by jaemoney135** [Berkanktk/CyberSecurity#5] [Q&A] [answered]
I am a CIS major. I graduate this summer. My goal is to get into cyber security. Maybe finding vulnerabilities in companies or defending I'm
Not sure yet. Can someone recommended any books that gives tips and tricks or something like a study guide so I can do somenhandsnon work. Maybe books in how to get around firewalls or build one myself. Also if anyone has any suggestions or guide me better on what might be better for one to study to make some good decent money.

**Comment by Berkanktk** [comment, marked as answer]
Maybe this can give you a start? 😊 https://github.com/Berkanktk/CyberSecurity/discussions/4#discussioncomment-6781175. Other than that i can recommend reading/watching these channels:
* https://primer.picoctf.com/
* https://book.hacktricks.xyz/
* https://www.youtube.com/@NetworkChuck
* https://www.youtube.com/@_JohnHammond/

I would definitely also practice on https://play.picoctf.org/, as they have some very beginner friendly challenges and read my Wiki, while also looking into the stuff in the [/More ](https://github.com/Berkanktk/CyberSecurity/tree/master/More)directory

> what might be better for one to study to make some good decent money.

If you are only interested in the field because of the money, then you should maybe re-evaluate if you also are willing to invest a lot of time and nights into studying. The field is big, and you will never be able to know everything. I'm saying this because the most money probably lies on the most difficult skillsets. 

And don't take me wrong, but the number one thing in IT, is to being able to Google. You are simply asking a question on a page that not many people will see, while we today have AI and huge open source resources available online to everyone. First Google Dorking, then the basics of Computer Science (which i assume you have based on your CIS major) and then jump into Cybersecurity. Wish you good luck! 😄

**Reply by jaemoney135** [reply]
Yes! I'm learning about google dorking. I visited Shodan and netcraft. I got a really cool book  that has things that a lot of people would be like 😮🤯 today I did some side work for security like pinging and Enum commands etc etc. google is literally your best friend in this field. I find the or , && commands really neat. I also been doing some swift because a friend of mine said to maybe start a app if I want to make some money. What I really want to do is build a block chain app that would be sooooo goood but I need a team for this. I also been learning a little smart contract coding and high freq coding. Today I did some testing on the Kali website from  a book that I got from "occupy the web" im a little all over the place but it is what it is. it's not really for the money honestly because I am finding my nights with my butt aching from hours on my PC just drenching myself in knowledge. I'm also familiar with the pico websites, but haven't done anything on their yet but PWD lol . I might not be ready for their tutorials but then again, that's what they are there for. I appreciate you answering me though. Much love and respect . Take care! 🫡

**Reply by jaemoney135** [reply]
Thank you for the hack tricks site looks really dope. And sorry for the late reply as well.

**Comment by dannyreew99-art** [comment]
what book did you find that was  😮🤯
